hail/python/hailtop/batch/backend.py

```diff
@@ -470,6 +475,7 @@ def _cp(src, dst):
 
             j = bc_batch.create_job(image=job._image if job._image else default_image,
                                     command=['/bin/bash', '-c', cmd],
```

Comment:
could override here instead of adding to spec

Reply:
Yes, that is much simpler.